zhcsshim.go

```diff
@@ -1,3 +1,5 @@
+// +build windows
```

Comment:
Does this contain windows only code? If so, we should keep it from compiling.

go generate should not affect the build tag.